tests/ipex/test_pipelines.py

```diff
@@ -144,10 +145,11 @@ def test_text_generation_pipeline_inference(self, model_arch):
             "text-generation", model_id, accelerator="ipex", torch_dtype=dtype, device_map=DEVICE
         )
         inputs = "Describe a real-world application of AI."
+        max_new_tokens = 10 if model_arch != "qwen2" else 2
```

Comment:
why only two tokens ? that's not enough to make sure generators match

Reply:
The IPEX OP has a little precision loss, it also appears in [test_logits](https://github.com/huggingface/optimum-intel/blob/main/tests/ipex/test_modeling.py#L114). So we cannot guarantee the output tokens are exactly the same since the logits are not exactly the same. 

Overall, I think the test is not very reasonable cause we never proposed that we have the exactly same output tokens as transformers.